Partition (WBP)/Functions/VI0049C.VTI0049.fun.yml

```diff
@@ -140,6 +141,48 @@
     
     Endroutine
     
+    *
+    * Execute SQL in LANSA and log the time
+    *
+    Mthroutine Name(mthExecuteSqlInLansa)
+    Define_Map For(*BOTH) Class(#BIFA002A) Name(#this_rc)
+    Define Field(#Start) Type(*DEC) Length(10) Decimals(3)
+    Define Field(#Finish) Type(*DEC) Length(10) Decimals(3)
+    Define Field(#Duration) Type(*DEC) Length(7) Decimals(0)
+    Define Field(#DB_Type) Type(*CHAR) Length(32)
+    Define Field(#LANSA) Type(*CHAR) Length(32)
+    Define Field(#LansaTime) Type(*CHAR) Length(32)
+    Def_List Name(#w_lLansaExecutionList) Fields(#DB_Type #LANSA #LansaTime) Type(*WORKING) Entrys(*MAX)
+    
+    #LF_rc := OK
+    #LANSA := ' ' + 'LANSA'
+    Case Of_Field(#LF_DB_Type)
+    When Value_Is(= MSSQLS = SQLAZURE = ODBCORACLE = MYSQL = SQLANYWHERE)
```

Comment:
There's no need for this when command. The only DB_TYPE at this point in the program are the ones listed.